myst_nbs/gaussian_processes/GP-MaunaLoa.myst.md

```diff
@@ -574,7 +581,24 @@ Also, using only historical CO$_2$ data may not be the best predictor.  In addit
 
 Next, we'll see about using PyMC3's GP functionality to improve the model, look at full posteriors, and incorporate other sources of data on drivers of CO$_2$ levels.
 
++++
+
+## Authors
+* Authored by Bill Engels in September, 2017 ([pymc#2444](https://github.com/pymc-devs/pymc/pull/2444))
+* Updated by Chris Fonnesbeck in December, 2020 ([pymc-examples#8](https://github.com/pymc-devs/pymc-examples/pull/8))
```

Comment:
```suggestion
* Updated by Chris Fonnesbeck in December, 2020
```

No need to add the PRs, and in this case the linked PR doesn't have any updates, it only restructures the untouched notebooks.